modules/benchmark/R/metric_timeseries_plot.R

```diff
@@ -13,7 +13,7 @@ metric_timeseries_plot <- function(metric_dat, var, filename = NA, draw.plot = i
   
   # Attempt at getting around the fact that time can be annual and thus as.Date won't work
   date.time <- try(as.Date(metric_dat$time), silent = TRUE)
-  if (class(date.time) == "try-error"){
+  if (inherits(date.time,"try-error")){
```

Comment:
Here (and elsewhere), please follow our style guide -- make sure there are spaces after commas and between close-parens and opening braces.

```suggestion
  if (inherits(date.time, "try-error")) {
```